app/views/posts/show.html.erb

```diff
@@ -8,18 +8,34 @@
   <div class="post">
     <h3><%= @post.title %> by <%= @user.name %></h3>
     <span>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></span>
+    <!-- likes button form -->
+    <%= form_with(model: @post, url: user_post_path, method: :post, local: true) do |form| %>
+      <%= form.hidden_field :user_id, value: @current_user.id %>
+      <%= form.hidden_field :post_id, value: @post.id %>
+      <%= form.submit 'Like' %>
+    <% end %>
     <p><%= @post.text %></p>
   </div>
   <div class="comments">
     <% if @comments.length>=1 %>
-          <% @comments.each do |comment| %>
-            <div class="comment">
-              <p><%= comment[:user] %>: <%= comment[:comment] %></p>
-            </div>
-          <% end %>
-          <% else %>
-            <p class="nocomments">No comments yet</p>
-          <% end %>
+      <% @comments.each do |comment| %>
+        <div class="comment">
+          <p><%= comment[:user] %>: <%= comment[:comment] %></p>
+        </div>
+      <% end %>
+    <% else %>
+      <p class="nocomments">No comments yet</p>
+    <% end %>
+  </div>
+  <!-- form for a new comment -->
+  <div class="new_comment">
+    <%= form_with(model: @comment, url: user_post_path, method: :post, local: true) do |form| %>
+      <span><%= @current_user.name %>: </span>
+      <%= form.text_field :text, placeholder: "Enter your comment here" %>
+      <%= form.hidden_field :user_id, value: @current_user.id %>
+      <%= form.hidden_field :post_id, value: @post.id %>
+      <%= form.submit 'Add Comment' %>
+    <% end %>
```

Comment:
- [ ] It would be better if the comment created on behalf of the current user is shown on the `users/1/post/1` users post show page as seen in the screenshot below
![Screenshot from 2023-06-25 06-33-08](https://github.com/Swarzstein/Blog-app/assets/46388113/e9049a8d-fb38-43f0-8b6e-ae31669093d1)